service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/ServiceClient.java

```diff
@@ -30,6 +30,8 @@
     protected IotHubConnectionString iotHubConnectionString;
     private IotHubServiceClientProtocol iotHubServiceClientProtocol;
 
+    private ProxyOptions proxyOptions;
```

Comment:
I don't see where this is being set. We seem to pass it into ServiceClient ctor though.

Reply:
And AmqpSend